lib/webdriver.js

```diff
@@ -110,7 +110,7 @@ Webdriver.prototype._init = function() {
     // retrieving session
     try{
       jsonData = JSON.parse(data);
-      _this.sessionId = jsonData.value ? jsonData.value.sessionId : false;
+      _this.sessionID = jsonData.value ? jsonData.value.sessionId : false;
```

Comment:
Hi, could this change be considered to be a bit of a breaking one, since it saves the correct value to the place it should've been saved "every-time" by previous intentions of the code written here.
But because the code used to be incorrect, it used to be saved into an incorrect place and the `jsonData.sessionId` is thus not used?

Reply:
hmm, maybe we should assign it to both spellings for now to be safe.